app/templates/partials/summary/sectionsummary.html

```diff
@@ -0,0 +1,54 @@
+{% set not_answered = 'No answer provided' %}
```

Comment:
Why cant we just use the summary.html?

Reply:
We would have to change the design to match the card and that summary.html is used elsewhere so best to make a separate file for section summaries

Reply:
I don't think its best to duplicate code.
what is the difference in the 2?

Reply:
Mainly title for the group being passed and text on the change answer link?

Reply:
I agree with @jonnyshaw89 — don't duplicate templates